lumen/command/app.py.jinja2

```diff
@@ -0,0 +1,43 @@
+import lumen.ai as lmai
+
+{# Import specified agents #}
+{%- if agents is defined -%}
+{% for agent in agents %}
+from lumen.ai.agents import {{ agent }}
+{%- endfor %}
+{%- endif %}
+
+{%- macro llm_init(llm_provider, api_key=None, model_kwargs=None) -%}
```

Comment:
Can you explain the point of the macro? Seems like a macro is only helpful if it's used more than once. Can you remove it?